2009/serial/C/bigdecimal.h

```diff
@@ -0,0 +1,64 @@
+#ifndef BIGDECIMAL_H_
+#define BIGDECIMAL_H_
+
+#include <deque>
+#include <string>
+
+using namespace std;
+
+class bigdecimal
+{
+private:
+    deque<int>* digits;
+    uint places, dot;

```

Comment:
I got the following error, while trying to compile it:
`error: unknown type name 'uint'; did you mean 'int'?`. Can you fix this, please?